docs/xls-70d-credentials/reference/deposit_authorized.md

```diff
@@ -0,0 +1,183 @@
+        "383D269D6C7417D0A8716B09F5DB329FB17B45A5EFDBAFB82FF04BC420DCF7D5"
+      ],
+      "deposit_authorized" : true,
+      "destination_account" : "rsUiUMpnrgxQp24dJYZDhmV4bE3aBtQyt8",
+      "ledger_hash" : "BD03A10653ED9D77DCA859B7A735BF0580088A8F287FA2C5403E0A19C58EF322",
+      "ledger_index" : 8,
+      "source_account" : "rEhxGqkqPPSxQ3P25J66ft5TwpzV14k2de",
+      "status" : "success",
+      "validated" : true
+   }
+}
+```
+{% /tab %}
+{% /tabs %}
+
+The response follows the [standard format][], with a successful result containing the following fields:
+
+| Field                  | Type                      | Required? | Description   |
+|:-----------------------|:--------------------------|:----------|:--------------|
+| `credentials`          | Array of [Hash][]         | No        | The credentials specified in the request, if any. |
+| `deposit_authorized`   | Boolean                   | Yes       | Whether the specified source account is authorized to send payments directly to the destination account. If `true`, either the destination account does not require [Deposit Authorization](https://xrpl.org/docs/concepts/accounts/depositauth) or the source account is preauthorized. |
+| `destination_account`  | String - [Address][]      | Yes       | The destination account specified in the request. |
+| `ledger_hash`          | String                    | No        | The identifying hash of the ledger that was used to generate this response. |
+| `ledger_index`         | Number - [Ledger Index][] | No        | The ledger index of the ledger version that was used to generate this response. |
+| `ledger_current_index` | Number - [Ledger Index][] | No        | The ledger index of the current in-progress ledger version, which was used to generate this response. |
+| `source_account`       | String - [Address][]      | Yes       | The source account specified in the request. |
+| `validated`            | Boolean                   | No        | If `true`, the information comes from a validated ledger version. |
+
+{% admonition type="info" name="Note" %}
+A `deposit_authorized` status of `true` does not guarantee that a payment can be sent from the specified source to the specified destination. For example, the destination account may not have a [trust line](https://xrpl.org/docs/concepts/tokens/fungible-tokens) for the specified currency, or there may not be enough liquidity to deliver a payment.
+{% /admonition %}
+
+## Possible Errors
+
+* Any of the [universal error types][].
+* `invalidParams` - One or more fields are specified incorrectly, or one or more required fields are missing.
+* `actMalformed` - An [Address][] specified in the `source_account` or `destination_account` field of the request was not properly formatted. (It may contain a typo or be the wrong length, causing a failed checksum.)
+* `badCredentials` - At least one of the supplied credentials doesn't exist, is expired, or has not been accepted.
```

Comment:
Just an interesting style choice that we use the contraction _doesn't_ here, but not in the next 3 lines.